proto/cork/v1/genesis.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package cork.v1;
+
+import "cork/v1/tx.proto";
+import "cork/v1/cork.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/cork/types";
+
+
+// GenesisState - all cork state that must be provided at genesis
+message GenesisState {
```

Comment:
Not sure it matters. The same exact group that would decide to change this would also be submitting transactions, with the same vote powers. With Allocation it was slightly different because it wasn't clear there would be a central data provider, it was possible the validators would need to analyze the state of the cellars on their own, and the params would tell them which ones to pay attention to. That's not the case with Cork